.github/policies/resourceManagement.yml

```diff
@@ -170,6 +170,23 @@ configuration:
         
         - [ ] Complete [localizability testing](https://github.com/NuGet/NuGet.Client/blob/dev/docs/ui-guidelines.md#localizability-testing).'
       description: 'Remind PR author to test accessibility and localizability'
+    - if:
+      - payloadType: Pull_Request
+      - isAction:
+          action: Opened
+      - filesMatchPattern:
+          pattern: '.*\.yml$'
+          matchAny: true
+      then:
+      - addReply:
+          reply: 'Since this PR is changing CI as it modifies YML files, please verify that you''ve addressed how this change should apply to release branches. Either: 
+          
+          - [ ] Link to equivalent PRs or Issues for each currently [supported branch](https://github.com/NuGet/Client.Engineering/blob/main/docs/HotSeat/NuGet.Client-Branches.md#supported-branches). 

```

Comment:
It doesn't make sense to me to create pull requests for servicing branches until after the first PR making the change is approved, to avoid needing to duplicate a lot of work keeping them all in sync when any one of them gets a comment requiring a change.

I expect that this was the intent, cherry pick the change and create servicing branch PRs after there are no more changes being requested, but this comment being added on PR open, and a pedantic read of the message sounds like it's asking for the PRs right away.

Perhaps `action: Opened` (line 176) could be changed to `action: Merged`? (I'm guessing that's the correct value)

Reply:
FYI I'm going to go ahead and merge this because the cost of people continuing to miss CI changes on servicing branches is very high. Just to be clear, we can iterate on the messaging, I'm just trying to get some reminder in ASAP. Feel free to comment on this issue if you have another specific suggestion https://github.com/NuGet/Client.Engineering/issues/3267